### ISSUE TITLE
DP 2.2.0 on Debian Bookworm

### ISSUE THREAD

**Issue by m00nglum** [dotproject/dotProject#205]
I'm having a heck of a time installing this on Debian Bookworm.  No matter what GD package I throw at it, it always fails to satisfy the installer.  What packages exactly are needed to satify this dependency?  Also, I have "put session.use_trans_sid Off" everywhere I can think of, my ~apache/php.ini, ~cli/php.ini my dp config.php, the .htaccess file in the root of my php install... The installer still always complains about it.

If I force the install without these two things not satisfied, then obviously my gannt charts don't work, but oddly enough, I don't have any project colors.  They all turn white.

Has anyone successfully installed this on Bookworm?

**Comment by fatabek** [comment]
Hello, during installation it checks the "session.use_only_cookies" parameter along with "session.use_trans_sid" (with logical comparison with "and" in the same if check). For this reason, even if you fix the "session.use_trans_sid" parameter, it continues to give errors because you do not fix the "session.use_only_cookies" parameter. So edit the "session.use_only_cookies" parameter before installation and try again.

**Comment by m00nglum** [comment]
You are saying fixing the session.use_only_cookies will fix the GD issue?  Are we talking about the same thing here?  I'm trying to find out what the dependencies are to get the Gannt chart going.. it needs GD, and I have several GD packages installed... but it doesn't seem happy with any of them.  Are you saying if I fix the "trans_sid" issue, Gannt charts will start working?


Thanks.

**Comment by fatabek** [comment]
No, I can't say anything about GD or gantt chart but what I mentioned is the solution for why the installer error persists.
